packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/Edge.java

```diff
@@ -27,8 +27,16 @@
 @Immutable
 public final class Edge implements IDiagramElement {
 
+    /**
+     * You should use SELECTED_NODES.
+     *
+     * @deprecated </a>
+     */
+    @Deprecated(forRemoval = true)
     public static final String SELECTED_EDGE = "selectedEdge";
 
+    public static final String SELECTED_EDGES = "selectedEdges";
```

Comment:
I prefer to have those variables grouped at one location instead of having them scattered across the codebase. I did not create a constant for that one on purpose.